doc/designs/quic-design/rx-depacketizer.md

```diff
@@ -0,0 +1,195 @@
+RX depacketizer
+===============
+
+This component takes a QUIC packet and parses the frames contained therein,
+to be forwarded to appropriate other components for further processing.
+
+In the [overview], this is called the "RX Frame Handler".  The name "RX
+depacketizer" was chosen to reflect the kinship with the [TX packetizer].
```

Comment:
Please edit the overview if you are changing the name.

Reply:
Yup, done